src/main/kotlin/app/revanced/patches/youtube/misc/video/speed/VideoSpeeds.kt

```diff
@@ -1,24 +0,0 @@
-package app.revanced.patches.youtube.misc.video.speed
```

Comment:
@oSumAtrIX This file is empty due to some reasons!

Reply:
Please open a follow up PR, seems like this was caused by the merge conflict resolution.